app/konnect/configure/runtime-manager/runtime-groups/manage.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Manage Runtime Groups
+no_version: true
+---
+
+Create, update, and delete runtime groups through the
+[Runtime Manager](https://konnect.konghq.com/runtime-manager) in Konnect.
+
+You can find the list of all runtime groups in the Runtime Manager. If you don't
+have an Enterprise account, the Runtime Manager will only ever have one
+runtime group entry.
+
+From a runtime group details page, you can
+[create and manage runtime instances](/konnect/configure/runtime-manager/#types-of-runtimes)
+and configure any [global entities](/konnect/configure/runtime-manager/manage-entities)
+in the group.
+
+## Prerequisites
+* TBA: Required permissions
+* TBA: Any prereqs for deleting the group
+
+## Create a runtime group
+{:.badge .enterprise}
+
+1. In Konnect, open the ![runtimes icon](/assets/images/icons/konnect/icn-runtimes.svg){:.inline .konnect-icn .no-image-expand}
+**Runtime Manager** from the left side menu.
+
+1. Click **Create Runtime Group**.
+
+1. Name the runtime group and enter an optional description.
+
+    Each runtime group in the organization must have a unique name.
+
+1. Add any labels in key-value pair format.
+
+    For example, you might set `location:us-west`, where `location` is the key
+    and the `us-west` is the value.
+
+    These labels are completely custom. Set anything that you need.
```

Comment:
Are there any limitations? Can I use 500 😄 emoji as my label?

Reply:
Question might be moot, as I don't know if labels are going to be available after all: https://github.com/Kong/docs.konghq.com/pull/3770#discussion_r835668611